test/dataframe.jl

```diff
@@ -51,17 +51,7 @@ module TestDataFrame
     @test names(dfc[1, :b]) == [:c, :e]
     @test names(dfdc[1, :b]) == [:c]
 
-    #
-
     x = DataFrame(a = [1, 2, 3], b = [4, 5, 6])
-    v = DataFrame(a = [5, 6, 7], b = [8, 9, 10])
```

Comment:
Still need to do something about this: https://github.com/JuliaData/DataFrames.jl/pull/1364/files#r171626211

Reply:
I did, it's in `deprecated.jl` now.  This dataframe `v` is removed here because it is no longer used in this file (appears again in `deprecated.jl`).